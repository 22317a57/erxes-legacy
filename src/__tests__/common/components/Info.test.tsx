import { mount, shallow } from 'enzyme';
import * as React from 'react';
import * as renderer from 'react-test-renderer';

import Info from '../../../modules/common/components/Info';

describe('Info component', () => {
  const defaultProps = {
    children: 'child'
  };

  test('renders Info successfully', () => {
    shallow(<Info {...defaultProps} />);
  });

  test('renders Info successfully with default value', () => {
    const wrapper = mount(<Info {...defaultProps} />);
    const props = wrapper.props();

    expect(props).toMatchObject(defaultProps);
  });

  test('snapshot matches', () => {
    const rendered = renderer.create(<Info {...defaultProps} />).toJSON();

    expect(rendered).toMatchSnapshot();
  });
});
