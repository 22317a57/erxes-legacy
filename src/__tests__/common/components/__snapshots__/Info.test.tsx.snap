// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Info component snapshot matches 1`] = `
<div
  className="sc-bdVaJa bvDBpf"
>
  child
</div>
`;
